webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -87,7 +87,6 @@ export default class AttachIssueModal extends PureComponent {
         const component = (
             <div>
                 <JiraIssueSelector
-                    id={'issue'}
```

Comment:
This was used purely as something to pass to `addValidate`. Since that function no longer requires a key to be used, this prop is useless.